src/components/AddBook/index.js

```diff
@@ -17,24 +16,22 @@ const AddBook = () => {
   const submitBookToStore = (e) => {
     e.preventDefault();
     const newBook = {
-      id: uuidv4(),
       title: e.target.title.value,
-      author: e.target.author.value,
+      category: e.target.category.value,
 
     };
 
     dispatch(addBook(newBook));
     e.target.title.value = '';
-    e.target.author.value = '';
+    e.target.category.value = '';
   };
 
   return (
     <FormWrapper id="add-books">
       <Headings>ADD NEW BOOK</Headings>
       <FormContainer onSubmit={submitBookToStore}>
         <AddBookInput placeholder="Book title" name="title" />
-        <AddBookInput placeholder="Book author" name="author" />
-        <CategoryInput placeholder="Under construction" />
+        <CategoryInput placeholder="Under construction" name="category" />
         <Button type="submit">ADD BOOK</Button>
       </FormContainer>
     </FormWrapper>
```

Comment:
- [ ] The `addBook` component is not rendering. It doesn't allow us to check if your code is working as the requirements said. Please, it would be better to fix it. 🔨 

![image](https://user-images.githubusercontent.com/79876959/145504629-423feafb-e95a-4d89-8257-6bc1f6f4d213.png)